### PR TITLE
WIP: Update files to (draft) release of 0.7.0

### LINKED ISSUE #17: Update StandardAPI constructor
To handle this change: https://github.com/ScottishCovidResponse/data_pipeline_api/pull/64

### PR DESCRIPTION
**Pull request by AlexRobson**
Closes #17 #16 

WIP while 0.7.0 is unrelease (poresently using master)

### CODE REVIEW

**Comment by sdl1** [comment]
@AlexRobson 0.7.0 has been released, this can probably be merged now?

File: deps/build.jl [hunk 1]
```diff
@@ -6,4 +6,4 @@ using Conda
 
 pip = joinpath(Conda.BINDIR, "pip")
 run(`$pip install pyyaml setuptools-scm`)
-run(`$pip install git+https://github.com/ScottishCovidResponse/data_pipeline_api@0.6.0`)
+run(`$pip install git+https://github.com/ScottishCovidResponse/data_pipeline_api#master`)
```
Comment (sdl1) [inline deps/build.jl]:
```suggestion
run(`$pip install git+https://github.com/ScottishCovidResponse/data_pipeline_api@0.7.0)
```

File: test/pyapi.jl [hunk 2]
```diff
@@ -10,10 +10,6 @@
             @test read_distribution(api, "parameter-read", "example-distribution") == Gamma(1.0, 2.0)
             # print(api.read_distribution("parameter-read", "example-samples"))  # expected to fail
```
Comment (sdl1) [inline test/pyapi.jl]:
```suggestion
@test read_samples(api, "parameter-read", "example-samples") == [1.0, 2.0, 3.0]
```

**Comment by sdl1** [comment]
Needs version bump

**Comment by AlexRobson** [comment]
Bumped version and running CI. If this works I will merge.

**Pull request merged**